nmosnode/registry.py

```diff
@@ -460,16 +460,24 @@ def list_resource(self, type, api_version="v1.0"):
                     ))
         return response
 
+    def _len_resource(self, type):
+        if not type in self.permitted_resources:
+            return RES_UNSUPPORTED
```

Comment:
It looks like you repeat this check? Exactly the same code appears on lines 472 and 473. It does seem slightly odd that a method for checking the length of something checks it's validity? I guess if we were being super "Pythonic" this shouldn't be "checked" at all, but instead you should just try accessing something at "type" and do a try catch to see if it fails.